pkg/paragliderpb/paraglider.proto

```diff
@@ -111,6 +124,16 @@ message AttachResourceResponse {
     string ip = 3;
 }
 
+message CheckResourceRequest {
```

Comment:
Thoughts on naming this `CheckFixResourceRequest`?  Because we use the same thing in `fixResource` function

Reply:
As discussed, let's put these in separate messages to follow convention, even if its redundant.